uuid/uuid.go

```diff
@@ -27,6 +27,21 @@ func IsAndroid(uuid string) bool {
 	return IsValidFast(uuid) && matchUuidRegexAndroid(uuid) && !isTest(uuid)
 }
 
+type IdentityType int
+
+const (
+	IdentityTypeOther IdentityType = iota
+	IdentityTypeIDFAOrAAID
+	IdentityTypeNoid
+)
+
+func GetIdentity(uuid string) IdentityType {
+	if IsNoid(uuid) {
+		return IdentityTypeNoid
+	}
+	return IdentityTypeIDFAOrAAID
```

Comment:
maybe this can be ponentialy impoved to check for valid idfa/aaid but I am not sure we need to do it now